helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -1,9 +1,9 @@
 {{- if eq .Capabilities.KubeVersion.Minor "21" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.0.17" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.0.18" }}
```

Comment:
FYI @feiskyer these are the values that need to be changed when we cut new releases of cloud-provider-azure. We want to keep the `imageTag` values commented out so that we don't provide a default. The `imageTag` configuration is mean to be a user-provided override when using a custom image reference.

Thanks for doing this work as part of the release! (I'm happy to follow releases and maintain this myself if you'd like, but also happy for you to do them and for me to observe/lgtm as appropriate).